File: Native/Test/Python/mr-value-transform.py
#
#  Copyright 2010-2012 Fabric Technologies Inc. All rights reserved.
#

import fabric

client = fabric.createClient()

print(
  client.MR.createValueTransform(
    client.MR.createConstValue('Integer', -42),
    client.KLC.createValueTransformOperator(
      "xfo.kl", "operator xfo(io Integer value) { value *= -2; }", "xfo"
      )
    ).produce()
  )

client.close()

File: Native/Test/Node/mr-value-transform.js
/*
 *  Copyright 2010-2012 Fabric Technologies Inc. All rights reserved.
 */

FC = require('Fabric').createClient();

console.log(
  FC.MR.createValueTransform(
    FC.MR.createConstValue('Integer', -42),
    FC.KLC.createValueTransformOperator(
      "xfo.kl", "operator xfo(io Integer value) { value *= -2; }", "xfo"
      )
    ).produce()
  );

FC.close();
